src/transport/SecureSessionMgr.cpp

```diff
@@ -194,6 +194,10 @@ CHIP_ERROR SecureSessionMgr::SendMessage(SecureSessionHandle session, PayloadHea
         *bufferRetainSlot = EncryptedPacketBufferHandle::MarkEncrypted(msgBuf.Retain());
     }
 
+    ChipLogProgress(Inet, "Send message of type %d and protocolId %d with MsgId:%08" PRIX32 " on exchange %d",
+                    payloadHeader.GetMessageType(), payloadHeader.GetProtocolID(), packetHeader.GetMessageId(),
```

Comment:
This is logging a struct (which is what `payloadHeader.GetProtocolID()` returns) as %d.  That's not a great idea (and will fail to compile once #7480 lands).  Please fix that and double-check that the other format specifiers here are right.

Reply:
We also need to update the log on receiving path after #7480 land. Track this with #7503